### PR TITLE
2022-12-05: Build mobile menu, small general fixes

### LINKED ISSUE #8: Develop navigation system
We need to develop the navigation menu!  

**Acceptance criteria:**
* Create a server-side data structure to store the menu structure
* Develop the navigation menu as a single template which can be placed on each page
  * Ensure the navigation menu has a mobile version

**Requires:**
* We can store and alter data in the backend #7

### PR DESCRIPTION
**Pull request by some-molasses**
dropdown

Adds a mobile menu for small screen widths.  Also includes some general improvements, including:
 * Small changes to the navigation system
 * Addition of a favicon, fonts, and more colours
 * Small homepage tweaks

Resolves #8

### CODE REVIEW

File: public/assets/sass/carousel.scss [hunk 1]
```diff
@@ -1,12 +1,10 @@
-* {box-sizing:border-box}
 /* Slideshow container */
 .slideshow-container {
-  box-sizing:border-box;
+  box-sizing: border-box;
```
[the remark this answers is not in the document]
Comment (some-molasses) [reply public/assets/sass/carousel.scss:3]: Yes and yes! :)

**Comment by some-molasses** [comment]
> Could you put all the icons in the img folder in a subdirectory called 'icons' or something like that?

done!

File: server/config/navbar.json [hunk 2]
```diff
@@ -1,12 +1,6 @@
 [
-    {
```
[the remark this answers is not in the document]
Comment (ad-world) [reply server/config/navbar.json:2]: no this is fine, good stuff

**Review by ad-world — APPROVED** [review]
LGTM :) Thanks for the changes

**Pull request merged**